src/main/java/mate/academy/springboot/swagger/util/SortOrderUtil.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortOrderUtil {
+    private static final int ORDER_DIRECTIONAL_INDEX = 1;
+    private static final int FIELD_INDEX = 0;
+
+    public Sort sort(String sortBy) {
+        List<Sort.Order> orders = new ArrayList<>();
+        if (sortBy.contains(":")) {
+            String[] sortingFields = sortBy.split(";");
+            for (String field : sortingFields) {
+                Sort.Order order;
+                if (field.contains(":")) {
+                    String[] fieldsAndDirections = field.split(":");
```

Comment:
Spring values can also be replaced with constants

Reply:
I'm not sure that I understood correctly. If I make this variable final, there will be a compile error